addons/parachute/functions/fnc_checkCutParachute.sqf

```diff
@@ -0,0 +1,18 @@
+#include "script_component.hpp"
+/*
+	Name: check Cut Parachute
+
+	Author: joko // Jonas
+
+	Description:
+	Check if Parachute can be Cuted
+
+	Parameters:
+	0: Object
+
+	Returns:
+	Boolean
+*/
```

Comment:
Incorrect function header (`XEH_respawn.sqf` has the correct format). Also tabs.